build/version.props

```diff
@@ -2,5 +2,26 @@
 
   <PropertyGroup>
     <IISAdministrationVersion>2.2.1</IISAdministrationVersion>
+    <MicroBuildCoreVersion>0.3.0</MicroBuildCoreVersion>
+    <SystemSecurityCryptographyX509CertificatesVersion>4.3.2</SystemSecurityCryptographyX509CertificatesVersion>
+    <SerilogExtensionsLoggingVersion>2.0.2</SerilogExtensionsLoggingVersion>
+    <SerilogVersion>2.7.1</SerilogVersion>
```

Comment:
Did you do an OSS registration for all the updated third-party dependencies?

Reply:
I did not do it myself. I am going to double check these OSS dependencies